### PR TITLE
Bookstore: connect to API

### PR DESCRIPTION
**Pull request by TadesseDev**
# Change introduces 😎
---
- Add `redux-thunk` to my project. ✔
- Use [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) for making API calls. ✔
- Fetch book list from Bookstore API on Books page load. ✔
- Refactor my add book and remove book features to persist changes in the server. ✔
- Add books and remove books' work in the same way after the refactor. ✔

> ## Note
> This pull request does not concern styles and updating the progress.

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
+import { useSelector, useDispatch } from 'react-redux';
 import Book from './sub-components/book';
 import AddNewBook from './sub-components/AddNewBook';
+import { getBooksFromServer } from '../redux/books/books';
 
 export default function Books() {
+  const dispatch = useDispatch();
   const books = useSelector((store) => store.books);
+  useEffect(() => {
+    // console.log('data');
```
Comment (brytebee) [inline src/components/Books.js]: - [ ] **Optional:** Kindly remove commented code. It is not professional to push commented code.

**Review by brytebee — APPROVED** [review]
Hi, @Tadesse-Alemayehu👋, 

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TadesseDev** [comment]
### Dear @brytebee, thak you for the review. I'll make sure to correct optional suggestions before making the merge. 👍

**Comment by TadesseDev** [comment]
### @brytebee, Optional suggestions is completed in https://github.com/Tadesse-Alemayehu/Bookstore/pull/4/commits/d3a2c2d93f94bb0564559494a1a27becd35634b8 commit hash

**Pull request merged**